examples/jenkins-master/jenkins-master-template.json

```diff
@@ -0,0 +1,251 @@
+{
+  "kind": "Template",
+  "apiVersion": "v1",
+  "metadata": {
+    "name": "jenkins-master",
+    "annotations": {
+      "description": "Jenkins service with Kubernetes plugin, without persistent storage. WARNING: Any data stored will be lost upon pod destruction. Only use this template for testing",
```

Comment:
Yes we do need rebuild the Jenkins image to install the sample job and GIT plugin.